notebooks/equivalence_checker/equivalence_checker_example.ipynb

```diff
@@ -0,0 +1,100 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Import"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from mqt.problemsolver.equivalence_checker import equivalence_checker"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Testing different Parameter Combinations for a Miter with known Counter Examples"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "num_bits = 6\n",
+    "num_counter_examples = 5\n",
```

Comment:
They are needed by the function "equivalence_checker.create_condition_string()"  in that cell. But since they are also required in the cell below, I additionally copied them there.

Reply:
Done